terraform/deployments/eks/main.tf

```diff
@@ -21,7 +21,7 @@ module "eks" {
   source  = "terraform-aws-modules/eks/aws"
   version = "17.1.0"
 
-  cluster_name    = "govuk-tmp" # TODO: name
+  cluster_name    = "govuk"
```

Comment:
This was on my TODO list so thanks for taking care of it! @kerin might have opinions on the name, given what we discussed about the "infrastructure" layer being application-agnostic.

Reply:
It would be good to use an ENV value. ie GOVUK-{ENV}.

Reply:
I think there will be one cluster per AWS account?

Reply:
Yeah, one cluster per aws account is the plan.